src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Don't extend classes to set behavior for them. Consider using interfaces to define behaviors that can be shared across multiple classes.